ziti/internal/edge_impl/conn.go

```diff
@@ -311,7 +316,21 @@ func (conn *edgeConn) Read(p []byte) (int, error) {
 }
 
 func (conn *edgeConn) Close() error {
-	return conn.close(false)
+	event := &closeConnEvent{
+		conn:        conn,
+		remoteClose: false,
+		errorC:      make(chan error, 1),
+	}
+	conn.msgMux.Event(event)
+	select {
+	case err, ok := <-event.errorC:
+		if ok && err != nil {
```

Comment:
is it possible to not be ok and have an err?

Reply:
It's redundant. I cleaned it up